src/bioregistry/data/bioregistry.json

```diff
@@ -86670,6 +86679,19 @@
       "orcid": "0000-0003-4423-4370"
     }
   },
+  "pephub": {
+    "contact": {
+      "email": "nsheffield@virginia.edu",
+      "name": "Nathan Sheffield",
+      "orcid": "0000-0001-5643-4068"
+    },
+    "description": "PEPhub is a database, web interface, and API for sharing, retrieving, and validating sample metadata. PEPhub uses Portable Encapsulated Projects (PEP) biological metadata standard to store, edit, and access PEPs in one place.",
+    "example": "gse185244",
+    "homepage": "https://pephub.databio.org/",
+    "name": "PEPhub",
+    "pattern": "^gse[0-9]{6}$",
+    "uri_format": "https://pephub.databio.org/geo/$1"
```

Comment:
Given how this resource is structured, I think we need to curate the `uri_format` as `https://pephub.databio.org/$1`, and allow for an identifier pattern that accepts any `x` or `x/y` combination (and not only gse* which is not generally the pattern for this resource). Then we can curate as example `geo`, and as extra example_extras `geo/gse185244`.